internal/common/helper.go

```diff
@@ -27,3 +28,16 @@ func RandomizeDuration(variance, duration float64) time.Duration {
 		int64(lowerLimit),
 		int64(upperLimit)))
 }
+
+func FormatRootHost(rootHost string) string {
+	formatted := strings.Replace(rootHost, "*", "w", 1)
+	if len([]byte(formatted)) > 63 {
+		formatted = string([]byte(formatted)[:63])
+	}
+
+	for []byte(formatted)[len([]byte(formatted))-1] == []byte(".")[0] {
+		formatted = string([]byte(formatted)[:len([]byte(formatted))-2])
+	}
+
+	return formatted
+}
```

Comment:
As discussed I'm going to remove this in favour of using a hashed value of the root host